.github/workflows/acceptance.yml

```diff
@@ -131,35 +188,22 @@ jobs:
         with:
           snuba: true
 
-      - name: yarn cache
-        uses: actions/cache@v2
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      - name: Wait for frontend build
+        uses: getsentry/action-wait-for-check@v1.0.0
+        id: wait-for-frontend
```

Comment:
This polls for the webpack job to complete so that we can run acceptance + webpack jobs in parallel.